### DISCUSSION TITLE
Add OpenSSF Scorecard to Maven and the mostly used plugins

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#8] [Ideas for Funding]
The OpenSSF Scorecard are an open standard to give a good indication about the security of the project and the build supply chain of the project. We should add that scorecards to all important projects and define a minimum score that must be reached in all projects.

**Comment by hboutemy** [comment]
if we could find a better way to name Maven vs Maven core vs Maven Plugins https://maven.apache.org/scm.html

**Comment by ascheman** [comment]
This discussion has been converted to Epic #69 for tracking.

Closing this discussion.
